engine/src/test/java/org/camunda/bpm/engine/test/bpmn/deployment/BpmnDeploymentTest.java

```diff
@@ -161,6 +169,10 @@ public void shouldLogWarningForDuplicateFilteringWithoutName() {
     
     // then
     assertEquals(1, loggingRule.getFilteredLog(CMD_LOGGER, "Deployment name set to null. Filtering duplicates will not work properly.").size());
+
+    // cleanup
+    String deploymentId = repositoryService.createDeploymentQuery().singleResult().getId();
+    repositoryService.deleteDeployment(deploymentId);
```

Comment:
Since we use the `testRule.deploy(...)`, we don't need to clean up here if I am not mistaken?
```suggestion
```